CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [XOSTOR] Various UI modifications in order to prepare XOSTOR official release (PR [#7568](https://github.com/vatesfr/xen-orchestra/pull/7568))
```

Comment:
This does not provide any information to users, either make it more explicit if you think that's relevant or remove it.